documentation/Migration/Update-1.0.0-rc1.md

```diff
@@ -261,7 +370,14 @@ There are some changes made to interfaces that generally don't need to be implem
 
 ### ICustomDataProcessor
 
-- Adds `TryEnableTable` method. This is similar to `EnableTable`, but catches most exceptions and returns a `bool`.
+- The following members have been removed:
+
+    * method `ICustomDataProcessor.BuildMetadataTables`
+
+- The following members have been added:
+    - method `TryEnableTable`. This is similar to `EnableTable`, but catches most exceptions and returns a `bool`.
+
+- `EnableTable` and `ProcessAsync` may now throw an `InvalidOperationException` if `ProcessAsync` has already been called.
 
 ### IDataProcessorExtensibilitySupport
```

Comment:
>IDataProcessorExtensibilitySupport [](http://example.com/codeflow?start=4&length=34)

IDataProcessorExtensibilitySupport has now been removed as it was only around to support internal tables.

Reply:
thanks!